.travis.yml

```diff
@@ -1,13 +1,17 @@
 language: ruby
 rvm:
   - ruby-head
-  - 2.2
-  - 2.1
-  - 2.0.0
-  - 1.9.3
-  - jruby-19mode
-gemfile:
-  - gemfiles/rails4.2.gemfile
-  - gemfiles/rails4.1.gemfile
-  - gemfiles/rails4.0.gemfile
-  - gemfiles/rails3.2.gemfile
```

Comment:
If you set it up this way then you can remove all the gemfiles in the test directory.